sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientTests.cs

```diff
@@ -98,6 +99,23 @@ public void ConstructorRequiresTheOptions()
             Assert.Throws<ArgumentNullException>(() => new FormRecognizerClient(endpoint, keyCredential, null));
         }
 
+        [Test]
+        public async Task FormRecognizerClientThrowsWithInvalidEndpoint()
```

Comment:
Wasn't sure about where to put this test. It's live, but we are not accessing actual Azure resources (since endpoint is fake). We can't record it because the request never gets a response. So I decided to leave it here, in the `FormRecognizerClientTests` file. Same reasoning applies for `FormTrainingClientThrowsWithInvalidEndpoint`.

Reply:
could you explain the purpose of this test?

Reply:
We are making sure the SDK can properly handle a scenario in which the user passes a non-existing FR resource endpoint to the client.